VERSION

```diff
@@ -0,0 +1 @@
+0.2-dev
```

Comment:
Once we work through issues with this PR, I plan to tag as 0.2. Let me know of any guidelines around version numbers.

Reply:
policy doc here: https://wiki.edgexfoundry.org/pages/viewpage.action?pageId=21823969

Reply:
Note that we're moving away from having VERSION files in the repository - instead the version information is generated at build time based on the most recent tag. Have a look at https://github.com/edgexfoundry/device-grove-c/blob/master/Makefile to see the sort of thing.

Reply:
I am working on an update to use the new versioning mechanism, and I'm confused about the git version tag. For example, commit edgexfoundry/device-grove-c@ee944e5eaf0fdadf041172ba91253cadf7663a84 is tagged as `v1.2.1`. Inclusion of the letter 'v' means the tag for the Docker image created by the Makefile `docker` target also includes the 'v' . For device-coap-c, that means the image tag will be `edgexfoundry/docker-device-coap-c:v0.2-dev`. Historically Docker image tags have not included the 'v'. So, should I just format the git version tag as device-grove-c does and be happy?

Reply:
That's a good point. We should probably `sed` out the 'v' when generating VERSION in the Makefile.
(The official release mechanisms use other means to generate VERSION, so the published images are tagged without the 'v').
Other services default to a version of `0.0.0` in the Makefile, I was trying to do better :)

Reply:
Sounds good; added pipe to sed to generate VERSION.